packages/compartment-mapper/test/fixtures-0/node_modules/excalibur/src/sword.js

```diff
@@ -0,0 +1,2 @@
+const { sword } = require('..');
+exports.sword = sword;
```

Comment:
this test was way better when i first wrote it as `export { sword } from (stone)` but alas its only relevant for commonjs behavior